src/account/TransferTransaction.js

```diff
@@ -8,6 +8,9 @@ import Long from "long";
 import TokenTransferMap from "./TokenTransferMap.js";
 import HbarTransferMap from "./HbarTransferMap.js";
 import TokenNftTransferMap from "./TokenNftTransferMap.js";
+//eslint-disable-next-line
```

Comment:
unused import

Reply:
more specifically:   12:8  warning  'NftId' is defined but never used  @typescript-eslint/no-unused-vars

Reply:
Ah, you need to import the type instead of the class itself. You should use the ``` * @typedef {import("../transaction/TransactionId.js").default} TransactionId
``` style imports instead.